index.html

```diff
@@ -130,7 +130,7 @@ <h3 class="profesional-art nocont">Profesional Art Printing Data</h3><h3 class="
             <li class="tags-li">Ruby</li>
           </ul>
         </div>
-        <button class="see-project non-content button-animations"> See Project</button>
+        <button class="see-project non-content button-animations btnpop"> See Project</button>
       </div>
       <div class="card custom-card2">
```

Comment:
[Optional] From the figma design(desktop version), the card  having a "See Project" button represents how each of the project cards should look when hovered while the ones without it represent how the cards should look normally. I suggest that you implement this in your project cards.

Normal card:
![image](https://user-images.githubusercontent.com/20554583/216478505-e01296a1-779b-428e-8d52-bd19288b3ba4.png)

Card when hovered:
![image](https://user-images.githubusercontent.com/20554583/216478531-63cc822b-b84a-4d4e-bc49-e2d3501fe9d9.png)